### PR TITLE
COMMERCE-11492 Drop DDM 5/5 - Clean up

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/COMMERCE-11492

Do not send before https://liferay.atlassian.net/browse/COMMERCE-9619 gets merged.

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
@luca-pellizzon @victoroscuro do not runt tests before 9619 gets merged. Thank you!

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/option/CommerceOptionType.java [hunk 1]
```diff
@@ -21,6 +21,8 @@ public interface CommerceOptionType {
 
 	public String getLabel(Locale locale);
 
+	public boolean hasValues();
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/option/CommerceOptionType.java:24]: What is this for?

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java [hunk 2]
```diff
@@ -650,6 +643,20 @@ public boolean hasDirectReplacement(CPSku cpSku) throws Exception {
 		return false;
 	}
 
+	@Override
+	public boolean hasMultipleCPSkus(CPCatalogEntry cpCatalogEntry) {
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java:647]: This is a helper for the content web module and it refers to the DTOs that we have in the front store. In fact, we are passing a CPCatalogEntry which is the DTO for the product definition and CPSku is the one used for CPInstance

File: modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/option/CommerceOptionType.java (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/option/CommerceOptionType.java:24]: This is to determin if the option type have values or the type does not accept them

File: modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPOptionsImporter.java [hunk 3]
```diff
@@ -94,15 +94,15 @@ private CPOption _importCPOption(
 		Map<Locale, String> descriptionMap = Collections.singletonMap(
 			locale, jsonObject.getString("description"));
 
-		String ddmFormFieldTypeName = jsonObject.getString(
-			"ddmFormFieldTypeName", "select");
+		String commerceOptionTypeKey = jsonObject.getString(
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPOptionsImporter.java:97]: Note to self: Check importer JSON files

**Review by luca-pellizzon — APPROVED** [review]

File: modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPOptionsImporter.java (hunk 3 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPOptionsImporter.java:97]: Everything's OK

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:reopen

**Pull request closed without merging**